setup.py

```diff
@@ -11,12 +11,12 @@
     warnings.warn("You are using pip version {}. However, pip version > 10.0.1 is required. If you continue with installation it is likely that you will face an error. See https://github.com/precice/python-bindings#version-of-pip3-is-too-old".format(pip.__version__))
 
 from setuptools import setup
+from setuptools import Command
 from setuptools.command.test import test
+from setuptools.command.install import install
```

Comment:
Just realized that this version now uses `setuptools.command.install` and not `distutils.command.install`. This makes `--single-version-externally-managed` is available (`python3 setup.py install --help`). See #64.

Reply:
Sounds good. I will check if it works with the standard installation procedure of Spack now.